### PR TITLE
[Compute] Updating 2019-03-01 schemas for Compute(dedicated host api)

### PR DESCRIPTION
**Pull request by mabhard**

### CODE REVIEW

**Comment by hyonholee** [comment]
LGTM

**Comment by hyonholee** [comment]
@antmarti-microsoft   Please review this PR.

**Comment by antmarti-microsoft** [comment]
Should these new resource types be added to the root schemas:
schemas\2014-04-01-preview\deploymentTemplate.json
schemas\2015-01-01\deploymentTemplate.json
schemas\2019-04-01\deploymentTemplate.json

Please could you add/update tests for new/changed resource types.

**Comment by mabhard** [comment]
The files already contain the resources types for version "2019-03-01".

File: schemas/2019-03-01/Microsoft.Compute.json [hunk 1]
```diff
@@ -259,6 +384,153 @@
...
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Availability Zone to use for this host group � only single zone is supported. The zone can be assigned only during creation. If not provided, the group supports all zones in the region. If provided, enforce each host in the group is in the same zone."
```
Comment (huangpf) [inline schemas/2019-03-01/Microsoft.Compute.json]: There seems to be a non-ASCII character in this line? Is it OK?

group � only

Comment (hyonholee) [reply schemas/2019-03-01/Microsoft.Compute.json]: @mabhard  Can you remove the character in this PR?  It looks like we need to update the Swagger by removing it too.

Comment (mabhard) [reply schemas/2019-03-01/Microsoft.Compute.json]: Removed.
Should I remove in swagger too?

**Pull request merged**